src/status_im/ui/screens/keycard/pin/views.cljs

```diff
@@ -94,9 +94,10 @@
       [react/view
        {:style {:flex-direction :row}}
        [checkbox/checkbox
-        {:checked?        save-password?
-         :style           {:margin-right 10}
-         :on-value-change #(re-frame/dispatch [:multiaccounts/save-password %])}]
+        {:checked? save-password?
+         :style    {:margin-right 10}}]
+       ;; should be reimplemented
```

Comment:
same here, can we have proper issues to track instead?

Reply:
/ just delete the commented code

Reply:
same here no issue, keycard functionality will be reimplemented with new designs

Reply:
i want to keep useful comments for developer who will be reimplementing keycard, just to save their time